src/ranking.jl

```diff
@@ -30,15 +30,23 @@ function ordinalrank!(rks::RealArray, x::RealArray, p::IntegerArray)
     return rks
 end
 
+
+"""
+    ordinalrank(x) -> Array
+
+Return the ordinal ranking of a real-valued array. Ordinal ranking,
+aka "1234" ranking, uses the literal sorted order. All items in `x`
```

Comment:
I don't understand "literal sorted order". I think you should make it clear that equal elements are given different (but successive) ranks based on their position in the input.